packages/Sandblocks-Smalltalk/SBPoint.class.st

```diff
@@ -31,15 +37,22 @@ SBPoint class >> newFor: aBlock [
 		type: Point
 ]
 
-{ #category : #'as yet unclassified' }
+{ #category : #'instance creation' }
 SBPoint class >> newForPoint: aPoint [
 
 	^ self new
 		x: (SBStNumber new contents: aPoint x asString) y: (SBStNumber new contents: aPoint y asString);
 		type: Point
 ]
 
-{ #category : #'as yet unclassified' }
+{ #category : #'instance creation' }
+SBPoint class >> newWithUnknown [
+
+	self flag: #broken.
+	^ self new addMorph: SBStMessageSend new
+]
```

Comment:
is this a utility function you needed somewhere in another project? I don't see any mention of it in the PR otherwise

Reply:
(I took the liberty of removing it while merging as it had no users and was marked #broken, but we can re-add it right away if needed somewhere :))

Reply:
Sounds fine. I do not even remember what this was for. :D